tests/unit/testsuite/Bolt/Boltpay/Model/ObserverTest.php

```diff
@@ -207,4 +215,56 @@ private function _createGuestCheckout($productId, $quantity)
 
         return $cart;
     }
+
+    /**
+     * Update Feature Switches if necessary
+     *
+     * event: controller_front_init_before
+     */
+    public function updateFeatureSwitches()
```

Comment:
I don't think this is used? If not, please remove.

Reply:
It is called via an event and is actually executed on every Magento request.

Reply:
Right. I'm specifically wondering if it needs to exist in the test class since it's already defined in Observer.php?

Reply:
oh, wow!  Good 👀 !!!

Reply:
Yup so @vitaliyreznikov please remove this.

Reply:
Thank you, Aden and Leon.  Done.